src-ui/app/models/ovr-device.ts

```diff
@@ -14,12 +14,37 @@ export type OVRDeviceRole =
   | 'Treadmill'
   | 'Stylus';
 
+export type OVRHandleType = 
+  | "HandPrimary"
+  | "HandSecondary"
+  | "Head"
+  | "Gamepad"
+  | "Treadmill"
+  | "Stylus"
+  | "FootLeft"
+  | "FootRight"
+  | "ShoulderLeft"
+  | "ShoulderRight"
+  | "ElbowLeft"
+  | "ElbowRight"
+  | "KneeLeft"
+  | "KneeRight"
+  | "WristLeft"
+  | "WristRight"
+  | "AnkleLeft"
+  | "AnkleRight"
+  | "Waist"
+  | "Chest"
+  | "Camera"
+  | "Keyboard"
+
 export interface OVRDevice {
   // Native properties
   battery: number;
   canPowerOff: boolean;
   class: OVRDeviceClass;
   role: OVRDeviceRole;
+  handleType: OVRHandleType;
```

Comment:
Do devices always have a role, or could there be a case where a device doesn't have one? If it's the latter, this should probably be optional.